client/src/components/CommentsUser.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+class CommentsUser extends React.Component {
```

Comment:
Refactor to export default at this point.
```suggestion
export default class CommentsUser extends React.Component {
```